lib/cordial/orders.rb

```diff
@@ -62,8 +62,8 @@ def self.create(id:, email:, purchase_date:, items:, link_id: '', mc_id: '')
         items: items
       }
 
-      body[:linkID] = link_id unless link_id.empty?
-      body[:mcID] = mc_id unless mc_id.empty?
+      body[:linkID] = link_id unless link_id&.empty? && !link_id.nil?
```

Comment:
Instead of doing this here, can we default them to `nil` instead of empty string? That should make this check a bit simpler. 

Also the `!` on `&& !link_id.nil?` is going to cast `link_id` into a boolean, which is going to result in some interesting and maybe not expected behaviour. 

```ruby
body[:linkID] = link_id unless link_id.nil?
```